packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+After the user's profile is retrieved from the SCIM server, Okta modifies the attributes that were changed and runs a PUT request with the new body to the `/users/${userId}` endpoint:
+
+    PUT /scim/v1/users/48e0a2da-0999-4f2c-87f4-80432cfe6617 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+        "userName": "test.user@okta.local",
+        "externalId": "00unrudb40SOK2eup0h7",
+        "name": {
+            "givenName": "Test1",
+            "familyName": "User"
+        },
+        "displayName": "Test1 User",
+        "locale": "en_US",
+        "active": true,
+        "emails": [{
+            "primary": true,
+            "value": "test.user@okta.local",
+            "type": "work"
+        }],
+        "meta": {
+            "created": "2019-10-18T06:50:23+00:00",
+            "lastModified": "2019-10-18T06:56:20+00:00",
+            "version": "a7fc04c68e3f5551e29a448031d45425"
+        },
+        "groups": []
+    }
+
+The response from the SCIM server needs to be the user’s SCIM profile:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 06:56:39 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "48e0a2da-0999-4f2c-87f4-80432cfe6617",
+        "userName": "test.user@okta.local",
+        "externalId": "00unrudb40SOK2eup0h7",
+        "name": {
+            "givenName": "Test1",
+            "familyName": "User"
+        },
+        "displayName": "Test1 User",
+        "locale": "en_US",
+        "active": true,
+        "emails": [{
+            "primary": true,
+            "value": "test.user@okta.local",
+            "type": "work"
+        }],
+        "groups": [],
+        "meta": {
+            "created": "2019-10-18T06:50:23+00:00",
+            "lastModified": "2019-10-18T06:56:39+00:00",
+            "version": "7c9d14512d03434910940e5574f6f8f0"
+        }
+    }
+
+### Update specific users (PATCH)
+
+<ApiOperation method="patch" url="/users/${userId}" />
+
+In some situations, you may want to send a PATCH operation to modify user attributes. For more information about modifying PATCH operations, see [RFC 7644, Section 3.5.2](https://tools.ietf.org/html/rfc7644#section-3.5.2). Okta can modify any mapped SCIM attributes through a PATCH operation.
+
+To update a user, the functionality needs to be enabled in Okta. In the Okta Admin Console, select the SCIM application from your list of applications. Under the **Provisioning** tab, click **To App** and select **Update User Attributes**.
+
+One important attribute modified through PATCH is the `active` attribute, representing the user’s current status.
+
+In order to deactivate a user, you need to enable the functionality in Okta. In the Okta Admin Console, under the **Provisioning** tab, click **To App** and select **Deactivate Users**.
```

Comment:
To deactivate users, you need to enable the functionality in Okta. In the Okta Admin Console, under the **Provisioning** tab, click **To App** and select **Deactivate Users**.